packages/SidePanel/src/SidePanel.js

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+import Dialog from "./components/Dialog";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import Overlay from "./components/Overlay";
+import Trigger from "./components/Trigger";
+import Group from "./components/Group";
+import FocusTrap from "./components/FocusTrap";
+import { extractChildren } from "./helpers";
+import useOffsetScroll from "./hooks/useOffsetScroll";
+import useBodyOverflow from "./hooks/useBodyOverflow";
+import useEscapeKey from "./hooks/useEscapeKey";
```

Comment:
Is one hook per file a rule?

Reply:
nop you can use multiple hooks in a file like:

const [user, setUser] = React.useState(null);
const [security, setSecurity] = React.useState(null);

Reply:
No, I mean the way `useEscapeKey` is the only hook exported from `hooks/useEscapeKey`.  This is preferred to `import { hook1, hook2 } from "./hooks/radicalHooks"`?